claim/README.md

```diff
@@ -116,6 +116,67 @@ For the connection process to work, the contents of `/var/lib/netdata` _must_ be
 restarts using a persistent volume.  See our [recommended `docker run` and Docker Compose
 examples](/packaging/docker/README.md#create-a-new-netdata-agent-container) for details.
 
+#### Known issue on systems with enabled seccomp
+
```

Comment:
Is this something that we can fix in upcoming releases? 
If yes, let's add a note that we are working on it. 
If not, and it always will require manual effort, I would explain why these hosts cannot be claimed.

Reply:
I think that it unlikely will be fixed in the upcoming release. cc @Ferroin @stelfrag

Reply:
> If not, and it always will require manual effort, I would explain why these hosts cannot be claimed.

That is a good idea and I had it in mind. The reason I didn't add - it may be very technical: `statx` system call used by the netdata-claim.sh script for file test operators is either not whitelisted in the Docker's default second profile ([commit](https://github.com/moby/moby/pull/36417/commits/b2a907c8cab64830c9e424eb1ec71e0429d326ff)  or not supported by libseccomp ([commit](https://github.com/seccomp/libseccomp/commit/5cbecad4e885d63ed6fd174b846b6aee09749cdc)) => this causes the system calls to return EPERM => some file test operators in the netdata-claim.sh script always fail.

Do we want to add something like this?

Reply:
I would add a note. The depth of how far we want to go in that explanation is a classic case of: ✨ Depends on our audience ✨ 

We could add something more superficial, for example: 

To use netdata on a system that has `seccomp` enabled, you need to claim your host manually. 
The default configuration of `seccomp` interferes with the automatic tests of the `netdata-claim.sh` script which is why you can't claim the host automatically. 

> Side note: Just to verify that I understood the context: We are on a node and we want to connect to a host, right? 
Is "claim" the host the right way to say it?

Reply:
I would recommend changing the title here to ‘Known issues on older hosts with seccomp enabled.’ It’s not a problem on a properly up-to-date host (for example, Alpine 3.15 and Fedora 35 as Docker hosts have no issues despite seccomp being enabled by default).